app/uk/gov/hmrc/plasticpackagingtax/registration/views/registration_page.scala.html

```diff
@@ -64,35 +70,41 @@ <h2 class="app-task-list__section">
 
     @pageTitle(text = messages("registrationPage.title"))
 
+    <h2 class="govuk-heading-s govuk-!-margin-bottom-2">Application @if(registration.isRegistrationComplete) { complete } else { incomplete }</h2>
```

Comment:
shouldn't we use paragraph from gds here?

Reply:
This has been addressed and rebased.
Tests added as well to test the new message